tests/scheduled_checks.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pytest
+import sys
+import requests
+
+sys.path.append('.')
+from simple.schema import *
+from astrodbkit2.astrodb import create_database, Database
+from sqlalchemy import except_, select, and_
+from . import REFERENCE_TABLES
+from scripts.ingests.utils import check_internet_connection
+
+DB_NAME = 'temp.db'
+DB_PATH = 'data'
+
+
+# Load the database for use in individual tests
+@pytest.fixture(scope="module")
+def db():
+    # Create a fresh temporary database and assert it exists
+    # Because we've imported simple.schema, we will be using that schema for the database
+
+    if os.path.exists(DB_NAME):
+        os.remove(DB_NAME)
+    connection_string = 'sqlite:///' + DB_NAME
+    create_database(connection_string)
+    assert os.path.exists(DB_NAME)
+
+    # Connect to the new database and confirm it has the Sources table
+    db = Database(connection_string, reference_tables=REFERENCE_TABLES)
+    assert db
+    assert 'source' in [c.name for c in db.Sources.columns]
+
+    # Load data into an in-memory sqlite database first, for performance
+    temp_db = Database('sqlite://', reference_tables=REFERENCE_TABLES)  # creates and connects to a temporary in-memory database
+    temp_db.load_database(DB_PATH, verbose=False)  # loads the data from the data files into the database
+    temp_db.dump_sqlite(DB_NAME)  # dump in-memory database to file
+    db = Database('sqlite:///' + DB_NAME, reference_tables=REFERENCE_TABLES)  # replace database object with new file version
+
+    return db
+
+
+def test_spectra_urls(db):
+    spectra_urls = db.query(db.Spectra.c.spectrum).astropy()
+    broken_urls = []
+    codes = []
+    internet = check_internet_connection()
+    if internet:
```

Comment:
I'm not familiar with `check_internet_connection` but if this returns False, the if statement doesn't run but the assert does and so this test would fail.